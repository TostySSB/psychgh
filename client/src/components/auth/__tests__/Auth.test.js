import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { BrowserRouter as Router } from "react-router-dom";
import { shallow, configure } from 'enzyme';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import {Login} from '../Login';
import {Register} from '../Register';

configure({adapter: new Adapter()});

function loginSetup() {
	const props = {
		loginUser: jest.fn(),
		auth: {
			isAuthenticated: false,
			loading: false,
			user: {}
		},
		errors: {}
	}
	const loginWrapper = shallow(<Login {...props}/>);
	return {props, loginWrapper};
}

describe('Login Component renders', () => {
	it('Login renders itself', () => {
		const { loginWrapper } = loginSetup();
		expect(loginWrapper.toJSON).toMatchSnapshot();
		
	})
});

function registerSetup() {
	const props = {
		
	}
}