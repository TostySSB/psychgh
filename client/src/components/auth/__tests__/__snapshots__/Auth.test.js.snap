// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Component renders Login renders itself 1`] = `undefined`;
